pkg/virt-controller/watch/migration/migration_test.go

```diff
@@ -314,16 +314,15 @@ var _ = Describe("Migration watcher", func() {
 		if len(vmi.Labels) == 0 {
 			vmi.Labels = nil
 		}
-		controller.vmiStore.Add(vmi)
-		key, err := virtcontroller.KeyFunc(vmi)
+		err := controller.vmiStore.Add(vmi)
 		Expect(err).To(Not(HaveOccurred()))
-		mockQueue.Add(key)
```

Comment:
Why is this not needed?

Reply:
This line makes no sense IIRC, it's adding a VMI key to the VMIM controller queue.